src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -261,3 +278,91 @@ To use the EKCO add-on to remove a node:
    The EKCO purge script For information about the EKCO purge script, see [Purge Nodes](/docs/add-ons/ekco#purge-nodes) in _EKCO Add-on_.
 
 1. Remove the node from the cluster.
+
+## Troubleshoot Node Removal
+
+This section includes information about troubleshooting issues with node removal in Rook Ceph clusters.
+
+### Rook Ceph v1.0.4 is Unhealthy with Mon Pod Pending
+
+#### Symptom
+
+After you remove a node from a Rook Ceph v1.0.4 cluster and you run `kubectl -n rook-ceph exec deployment.apps/rook-ceph-operator -- ceph status`, you see that Ceph is in an unhealthy state where a Ceph monitor (mon) is down. 
+
+For example:
+
+```
+health: HEALTH_WARN
+        1/3 mons down, quorum a,c
+```
+
+When you run `kubectl -n rook-ceph get pod -l app=rook-ceph-mon`, you see that the mon pod is in a Pending state. 
+
+For example:
+
+```
+NAME             READY  STATUS   RESTARTS  AGE
+rook-ceph-mon-a  1/1    Running  0         20m
+rook-ceph-mon-b  0/1    Pending  0         9m45s
+rook-ceph-mon-c  1/1    Running  0         13m
+```
+
+#### Cause
+
+This is caused by a known issue in Rook Ceph v1.0.4 where the rook-ceph-mon-endpoints ConfigMap still maps a node that was removed. For more information, see [Mon is never rescheduled](https://github.com/rook/rook/issues/2262#issuecomment-460898915) in the rook GitHub repository.
+
+#### Solution
```

Comment:
```suggestion
#### How to sort out the Ceph warning state to upgrade
```

Reply:
I think is better we avoid the word solution and goes in the direction of that is an workaround to came back to health state to upgrade which is the best option.

Reply:
Our solution, I think, is more to upgrade. 
ihmo would be better if we could avoid letting people think that is fine and that they should do that to solve the issue and continue to use this version. This is more like workarounds to return to the Ceph for a healthy state. It is not an official and tested operation for all possible edge-case scenarios. It is not either doc in the Rook docs.

Reply:
"Our solution, I think, is more to upgrade. ihmo would be better if we could avoid letting people think that is fine and that they should do that to solve the issue and continue to use this version" > The content that I wrote does tell people that the solution is that they must upgrade. To help reinforce this idea, I changed the title to "Workaround".